astropy/modeling/polynomial.py

```diff
@@ -985,8 +985,10 @@ class Polynomial2D(PolynomialModel):
     Parameters
     ----------
     degree : int
-        highest power of the polynomial,
-        the number of terms is degree+1
+        Polynomial degree: largest sum of exponents (:math:`i + j`) of
+        variables in each monomial term of the form :math:`x^i y^j`. The
+        number of terms in 2D polynomial of degree ``n`` is given by binomial
+        coefficient :math:`C(n + 2, 2) = (n + 2)! / (2!\,n!)`.
```

Comment:
If you want math notation here, I think you need to make the docstring a raw string.

Reply:
I think using raw string would fix your error:
```
E   SyntaxError: invalid escape sequence \,
```